### PR TITLE
BCDA-7089: Add home page updates

### PR DESCRIPTION
**Pull request by karinamzalez**
## 🎫 Ticket

https://jira.cms.gov/browse/BCDA-7089

## 🛠 Changes

- adds homepage updates to copy, reorganization of components, and image resizing. 

## ℹ️ Context for reviewers

- changes were made following design: https://confluence.cms.gov/display/BCDA/Home+Page+draft

## ✅ Acceptance Validation

- assert that home page matches design pdf
- screen recording:

Uploading Screen Recording 2023-08-15 at 11.02.29 PM.mov…

## 🔒 Security Implications

- [ ] This PR adds a new software dependency or dependencies.
- [ ] This PR modifies or invalidates one or more of our security controls.
- [ ] This PR stores or transmits data that was not stored or transmitted before.
- [ ] This PR requires additional review of its security implications for other reasons.

If any security implications apply, add Jason Ashbaugh (GitHub username: StewGoin) as a reviewer and do not merge this PR without his approval.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -25,6 +26,13 @@
     <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
       <img src="/assets/img/homepage/Static_Header.svg" alt="BCDA logo" class="ds-l-sm-col--6 ds-l-md-col--6 ds-l-lg-col--12">
     </div>
+    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
```
Comment (kyeah) [inline index.html:29]: I think the column sizes for the three columns need to shrink from 6 --> 4:

```suggestion
    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--4 ds-u-text-align--center">
```

Right now it pops down into a new row.

Screen Shot 2023-08-03 at 5 13 04 PM

File: index.html [hunk 2]
```diff
@@ -16,7 +16,8 @@
         THE RIGHT DATA TO THE RIGHT PEOPLE AT THE RIGHT TIME.
       </div>
       <div class="ds-u-font-size--base ds-u-padding-y--3">
-        The Beneficiary Claims Data API (BCDA) enables Accountable Care Organizations (ACOs) to retrieve Medicare Part A, Part B, and Part D claims data for their assigned beneficiaries. BCDA is currently serving ACOs participating in the Medicare Shared Savings Program and other eligible CMS Alternative Payment Models.
+        The Beneficiary Claims Data API (BCDA) gives organizations the insights they need to provide high quality, coordinated
```
Comment (kyeah) [inline index.html:19]: Assume this should stay centered, but in the wireframe it's left-aligned so would want to quickly confirm with Ali 😄

File: index.html (hunk 1 above)
Comment (kyeah) [inline index.html:29]: Also can we left-align the text for these bulleted items like in the wireframe?

File: index.html [hunk 3]
```diff
@@ -34,36 +42,78 @@
...
       <div class="ds-l-row">
         <div class="ds-l-col--12">
-          <div class="ds-u-font-size--h1 ds-u-color--white ds-u-font-weight--semibold"> How can ACOs benefit from BCDA? </div>
+          <div class="ds-u-font-size--h1 ds-u-color--white ds-u-font-weight--semibold"> How does BCDA benefit participants? </div>
         </div>
       </div>
       <div class="ds-l-row ds-u-margin-top--5">
         <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--4 benefit-prop">
-          <img src="/assets/img/homepage/ValueProp_1.svg" alt="Improve">
+          <img src="/assets/img/homepage/ValueProp_1.svg" width="50%" height="50%" alt="Improve">
```
Comment (kyeah) [inline index.html:50]: Should these icons be horizontally-centered now that they're smaller? they look a little awkward with left-alignment.

Screen Shot 2023-08-03 at 5 18 14 PM

File: index.html [hunk 4]
```diff
@@ -74,23 +124,27 @@ <h2 class="ds-u-font-size--h3 ds-u-font-weight--bold faq-question">
             1. Are Claim and Claim Line Feed (CCLF) files going away?
           </h2>
           <div class="ds-u-font-size--base ds-u-padding-top--1 ds-u-color--white">
-            No, CCLF files are not going away. BCDA is meant to supplement the data that ACOs already receive for the Shared Savings Program or other eligible model participation, which may include CCLF files. Please contact <a href="mailto:SharedSavingsProgram@cms.hhs.gov">SharedSavingsProgram@cms.hhs.gov</a> or your respective model with questions about CCLF files.
+            No, CCLF files are not going away. BCDA supplements the data that participants already receive for the Shared Savings Program (which may include CCLF files). Contact<a href="mailto:SharedSavingsProgram@cms.hhs.gov">SharedSavingsProgram@cms.hhs.gov</a> or your respective model with questions about CCLF files.
```
Comment (kyeah) [inline index.html:127]: Missing a space here!

```suggestion
            No, CCLF files are not going away. BCDA supplements the data that participants already receive for the Shared Savings Program (which may include CCLF files). Contact <a href="mailto:SharedSavingsProgram@cms.hhs.gov">SharedSavingsProgram@cms.hhs.gov</a> or your respective model with questions about CCLF files.
```

**Review by kyeah — COMMENTED** [review]
Thanks! 🌟

File: _updates/2023-05-16.html [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
+      release was designed as an expansion of the data available to Medicare providers from the BCDA and is available to
+      Accountable Care Organizations (ACOs) participating in the ACO Realizing Equity, Access, and Community Health (ACO
+      REACH) Model.
     </p>
+    <p><strong>
+        For more information on partially adjudicated claims, the User Guide can be found <a href="/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank" class="in-text__link">here</a> and the Data Dictionary can be found can be found <a href="https://confluence.cms.gov/assets/data/BCDA_Partially_Adjudicated_Claims_Enhancement_User_Guide_v1.0.0.pdf" target="blank"
+          class="in-text__link">here</a>.
+    </strong></p>
   </div>
 </div>
```
Comment (kyeah) [inline _updates/2023-05-16.html:17]: Oh this is part of the other ticket for the updates page, right? Can we remove this from the homepage updates? :D Thanks!

File: index.html (hunk 3 above)
Comment (laurenkrugen-navapbc) [reply index.html:50]: the icons are still centered-left; should they be?

File: _updates/2023-05-16.html (hunk 5 above)
Comment (karinamzalez) [reply _updates/2023-05-16.html:10]: @kyeah I think this is part of the home page updates

File: index.html [hunk 6]
```diff
@@ -25,6 +26,13 @@
     <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
       <img src="/assets/img/homepage/Static_Header.svg" alt="BCDA logo" class="ds-l-sm-col--6 ds-l-md-col--6 ds-l-lg-col--12">
     </div>
+    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--left">
+      <ul>
+        <li>Medicare Shared Savings Program ACOs</li>
+        <li>Kidney Care Choices (KCC)</li>
+        <li>ACO REACH</li>
```
[the remark this answers is not in the document]
Comment (karinamzalez) [reply index.html:33]: You are correct and this should be fixed.

File: index.html (hunk 3 above)
Comment (karinamzalez) [reply index.html]: They should be centered now!

File: index.html [hunk 7]
```diff
@@ -9,21 +9,31 @@
 ---
 
 <!-- Hero Banner -->
-<section role="region" aria-label="hero-banner" class="ds-l-container ds-u-padding--1 ds-u-sm-padding--2 ds-u-md-padding--3 ds-u-lg-padding--4 ds-u-xl-padding--5">
-  <div class="ds-l-row ds-u-align-items--center">
-    <div class="ds-l-sm-col--12 ds-l-md-col--12 ds-l-lg-col--6 ds-u-text-align--center">
-      <div class="ds-u-font-size--h1 ds-u-leading--base ds-u-font-weight--bold ds-u-padding-y--3 ds-u-text-transform--uppercase">
+<section role="region" aria-label="hero-banner" class="ds-l-container ds-u-padding--1 ds-u-sm-padding--2 ds-u-md-padding--3 ds-u-lg-padding--4 ds-u-xl-padding--5 ds-l-container">
```
Comment (laurenkrugen-navapbc) [inline index.html:12]: just checking in on this one, I know you and Ali did some coordination -- when I run the website locally, this part looks different from the wireframe, are we still going with the wireframe or did Ali give different feedback?

Running locally:
image

Wireframe:
image

Comment (karinamzalez) [reply index.html:12]: Hi @laurenkrugen-navapbc Ali suggested this change!

Comment (laurenkrugen-navapbc) [reply index.html:12]: Thanks for the context! 😄

**Review by laurenkrugen-navapbc — APPROVED** [review]

**Review by laurenkrugen-navapbc — DISMISSED** [review]

**Pull request merged**